.gitignore

```diff
@@ -138,3 +138,7 @@ dask-worker-space/
 # Test output
 tmp/
 .tmp_evaluation
+
+# private
```

Comment:
I think we dont want to have ignore files specific to a collaborator, so you could either use git to ignore changes made to these files like [this](https://git-scm.com/docs/git-update-index), or be careful when adding files to staging area

Reply:
Thanks, I will do that